Wrappers/Python/ccpi/framework/framework.py

```diff
@@ -1001,9 +994,41 @@ def min(self, *args, **kwargs):
     def max(self, *args, **kwargs):
         '''Returns the max pixel value in the DataContainer'''
         return numpy.max(self.as_array(), *args, **kwargs)
-
-    
-    
+        
+    def size(self):
+        '''Returns the number of elements of the DataContainer'''
+        return self.as_array().size
+    
+    # Logic operators between DataContainers and floats    
```

Comment:
Please add docstring to describe what the expectation of a comparison operator of a `DataContainer` with a float should be.